test/unit/aggregate_store_test.rb

```diff
@@ -85,13 +85,78 @@ def elasticsearch_store
         assert !passport.changed?
       end
 
-      should "respond to changed? appropriatetly when instance is not a active record object" do
+      should "respond to changed? appropriately when instance is not a active record object" do
         assert !@instance.changed?
 
         @instance.name = "blah"
         assert @instance.changed?
       end
 
+      Rails::VERSION::MAJOR > 4 && ['save', 'save!'].each do |save_method|
```

Comment:
Same comment

Reply:
Also, can we add a test that shows that `saved_changes?` gets reset to false on `passport.reload` and `@instance.reload`?